tests/api-resources/test_resources.py

```diff
@@ -82,6 +90,15 @@ def test_simple_flow(app, client, minimal_record, headers):
     data = res.json['hits']['hits'][0]
     data['metadata']['title'] = 'New title'
 
+    # Search it including UI metadata
```

Comment:
Minor: This should IMO be in a different test case. The simple flow tests is about the flow between different requests. The test is about the serialization which is being returned.

Reply:
Indeed I tried to squeeze it in the simple flow to avoid adding a separate case to keep the future testing time shorter but maybe it will be cleaner in a separate one.

Reply:
In this case I prefer separate tests, to make it more clear what is being tested. This will make maintenance easier in the future.